src/components/ItemsDisplay/ItemsDisplay.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const ItemsDisplay = () => {
+    return(
+        <div className='grid lg:grid-cols-4 lg:col-span-3 md:grid-cols-3 sm:grid-cols-2 grid-cols-1 gap-6' data-testid='itemsdisplay-div'>
+            {/* Add the items cards */}
+            {/* <p>1</p>
+            <p>1</p>
+            <p>1</p>
+            <p>1</p>
+            <p>1</p>
+            <p>1</p>
+            <p>1</p> */}
+        </div>
+    );
+};
+
+export default ItemsDisplay;
```

Comment:
is this tested with real cards or only with p's ?

Reply:
only with p the card are not done idk

Reply:
Yeah it is not but you can just copy a tailwind card component to test it out. P elements are inline and we cant know for sure if this will work fine or not by trying them only with p elements